### PR TITLE
[Docs] Update `bash` and `curl` blocks

### PR DESCRIPTION
**Pull request by pedrosousa**
Updates based on our guidelines:
* [Terminal prompts for `bash` blocks](https://developers.cloudflare.com/style-guide/formatting/code-block-guidelines/#for-bash-blocks)
* [Guidelines for `curl` commands](https://developers.cloudflare.com/style-guide/api-content-strategy/guidelines-for-curl-commands/)

### CODE REVIEW

File: content/cloudflare-for-platforms/cloudflare-for-saas/domain-support/hostname-validation/pre-validation.md [hunk 1]
```diff
@@ -81,27 +81,26 @@ To get and use the `ownership_verification` record:
...
     }
     ```
 
 3.  Have your customer place the `http_url` and `http_body` on their origin web server.
 
-    ```bash
+    ```txt
     ---
     header: Example response (truncated)
-    highlight: [7-10]
```
Comment (pedrosousa) [inline content/cloudflare-for-platforms/cloudflare-for-saas/domain-support/hostname-validation/pre-validation.md:104]: There are only three lines in the code block.

File: content/tenant/_partials/_create-account-api.md [hunk 2]
```diff
@@ -108,20 +108,20 @@ A request with a unit ID and KYC:
...
+--header "x-auth-key: <API_KEY>" \
+--header "Content-Type: application/json" \
+--data '{
+  "name": "<ACCOUNT_NAME>",
+  "type": "standard",
+  "business_name": "Cloudflare",
+  "business_email": "email@business.com",
+  "business_address": "San Francisco",
+  "business_phone": "1234567890",
+  "external_metadata": "{'\''testKey'\'': '\''testValue'\''}",
```
Comment (pedrosousa) [inline content/tenant/_partials/_create-account-api.md:122]: Escaped each `'` as `'\''`.

File: content/bots/concepts/sequence-rules.md [hunk 3]
```diff
@@ -24,26 +24,25 @@ Sequence rules is currently in private beta. If you would like to be included in
...
 ---
-
 curl --request PUT \
-  --url https://api.cloudflare.com/client/v4/zones/{zone_id}/fraud_detection/sequence_cookies \
-  --header 'Authorization: Bearer <API_TOKEN>' \
-  --header 'Content-Type: application/json' \
-  --data '{"enabled": true}'
+https://api.cloudflare.com/client/v4/zones/{zone_id}/fraud_detection/sequence_cookies \
+--header 'Authorization: Bearer <API_TOKEN>' \
+--header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/bots/concepts/sequence-rules.md]:
```suggestion
--header "Authorization: Bearer <API_TOKEN>" \
--header "Content-Type: application/json" \
```

File: content/cloudflare-for-platforms/workers-for-platforms/configuration/tags.md [hunk 4]
```diff
@@ -24,10 +25,11 @@ You can set a maximum of eight tags per script. Avoid special characters like `,
 You can include script tags and bindings on multipart script uploads in the metadata blob.
 
 ```bash
-curl -X PUT 'https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/workers/dispatch/namespaces/<NAMESPACE_NAME>/scripts/<SCRIPT_NAME>' \
--H 'X-Auth-Email: <EMAIL>' \
--H 'X-Auth-Key: <AUTH_KEY>' \
--H 'Content-Type: multipart/form-data' \
+curl --request PUT \
+'https://api.cloudflare.com/client/v4/accounts/{account_id}/workers/dispatch/namespaces/{namespace_name}/scripts/{script_name}' \
```
Comment (marciocloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/configuration/tags.md]:
```suggestion
"https://api.cloudflare.com/client/v4/accounts/{account_id}/workers/dispatch/namespaces/{namespace_name}/scripts/{script_name}" \
```

File: content/cloudflare-for-platforms/workers-for-platforms/configuration/tags.md [hunk 5]
```diff
@@ -24,10 +25,11 @@ You can set a maximum of eight tags per script. Avoid special characters like `,
...
+--header "X-Auth-Email: <EMAIL>" \
+--header "X-Auth-Key: <AUTH_KEY>" \
+--header 'Content-Type: multipart/form-data' \
```
Comment (marciocloudflare) [inline content/cloudflare-for-platforms/workers-for-platforms/configuration/tags.md]:
```suggestion
--header "Content-Type: multipart/form-data" \
```

File: content/cloudflare-one/_partials/access/_access-group.md [hunk 6]
```diff
@@ -29,9 +29,9 @@ Send a `POST` request to the [`/access/groups`](/api/operations/access-groups-cr
 
 ```bash
 curl https://api.cloudflare.com/client/v4/accounts/{account_id}/access/groups \
---header 'Content-Type: application/json' \
 --header 'X-Auth-Email: <EMAIL>' \
 --header 'X-Auth-Key: <API_KEY>' \
+--header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/cloudflare-one/_partials/access/_access-group.md]:
```suggestion
--header "X-Auth-Email: <EMAIL>" \
--header "X-Auth-Key: <API_KEY>" \
--header "Content-Type: application/json" \
```

File: content/cloudflare-one/_partials/gateway/_lists.md [hunk 7]
```diff
@@ -40,15 +40,15 @@ You can now use this list in the policy builder by choosing the _in list_ operat
 
 ```bash
 curl https://api.cloudflare.com/client/v4/accounts/{account_id}/gateway/lists \
---header 'Content-Type: application/json' \
 --header 'X-Auth-Email: <EMAIL>' \
 --header 'X-Auth-Key: <API_KEY>' \
+--header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/cloudflare-one/_partials/gateway/_lists.md]:
```suggestion
--header "X-Auth-Email: <EMAIL>" \
--header "X-Auth-Key: <API_KEY>" \
--header "Content-Type: application/json" \
```

File: content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md [hunk 8]
```diff
@@ -105,16 +107,16 @@ Using `jq` with the first option above, modify your script `add-multiple-zones.s
     printf "Adding ${domain}:\n"
 
     add_output=`curl https://api.cloudflare.com/client/v4/zones \
-      -H 'Content-Type: application/json' \
-      -H 'X-Auth-Email: <CLOUDFLARE_EMAIL>' \
-      -H 'X-Auth-Key: <API_KEY>' \
+      --header 'X-Auth-Email: <EMAIL>' \
+      --header 'X-Auth-Key: <API_KEY>' \
+      --header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md]:
```suggestion
      --header "X-Auth-Email: <EMAIL>" \
      --header "X-Auth-Key: <API_KEY>" \
      --header "Content-Type: application/json" \
```

File: content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md [hunk 9]
```diff
@@ -123,10 +125,9 @@ Using `jq` with the first option above, modify your script `add-multiple-zones.s
     printf "\n\n"
     printf "DNS quick scanning ${domain}:\n"
 
-    scan_output=`curl -X POST https://api.cloudflare.com/client/v4/zones/$domain_id/dns_records/scan \
-      -H 'Content-Type: application/json' \
-      -H 'X-Auth-Email: <CLOUDFLARE_EMAIL>' \
-      -H 'X-Auth-Key: <API_KEY>'`
+    scan_output=`curl --request POST https://api.cloudflare.com/client/v4/zones/$domain_id/dns_records/scan \
+      --header 'X-Auth-Email: <EMAIL>' \
+      --header 'X-Auth-Key: <API_KEY>'`
```
Comment (marciocloudflare) [inline content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md]:
```suggestion
      --header "X-Auth-Email: <EMAIL>" \
      --header "X-Auth-Key: <API_KEY>"`
```

File: content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md [hunk 10]
```diff
@@ -148,30 +149,28 @@ You can find your zones nameservers in the following locations:
     printf "Adding ${domain}:\n"
 
     add_output=`curl https://api.cloudflare.com/client/v4/zones \
-      -H 'Content-Type: application/json' \
-      -H 'X-Auth-Email: <CLOUDFLARE_EMAIL>' \
-      -H 'X-Auth-Key: <API_KEY>' \
+      --header 'X-Auth-Email: <EMAIL>' \
+      --header 'X-Auth-Key: <API_KEY>' \
+      --header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md]:
```suggestion
      --header "X-Auth-Email: <EMAIL>" \
      --header "X-Auth-Key: <API_KEY>" \
      --header "Content-Type: application/json" \
```

File: content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md [hunk 11]
```diff
@@ -148,30 +149,28 @@ You can find your zones nameservers in the following locations:
...
     printf "\n\n"
     printf "DNS quick scanning ${domain}:\n"
 
-    scan_output=`curl -X POST https://api.cloudflare.com/client/v4/zones/$domain_id/dns_records/scan \
-      -H 'Content-Type: application/json' \
-      -H 'X-Auth-Email: <CLOUDFLARE_EMAIL>' \
-      -H 'X-Auth-Key: <API_KEY>'`
+    scan_output=`curl --request POST https://api.cloudflare.com/client/v4/zones/$domain_id/dns_records/scan \
+      --header 'X-Auth-Email: <EMAIL>' \
+      --header 'X-Auth-Key: <API_KEY>'`
```
Comment (marciocloudflare) [inline content/fundamentals/setup/manage-domains/add-multiple-sites-automation.md]:
```suggestion
      --header "X-Auth-Email: <EMAIL>" \
      --header "X-Auth-Key: <API_KEY>"`
```

File: content/spectrum/about/load-balancer.md [hunk 12]
```diff
@@ -38,11 +38,11 @@ Under **Advanced health check settings**:
...
 ```bash
-curl 'https://api.cloudflare.com/client/v4/organizations/{ORG_ID}/load_balancers/monitors'  \
--H 'Content-Type: application/json' \
--H 'X-Auth-Email: user@example.com' \
--H 'X-Auth-Key: 00000000000' \
--X POST --data '{"description":"Spectrum Health Check","type":"tcp","port":2048,"interval":30,"retries":2,"timeout":5,"method":"connection_established"}'
+curl 'https://api.cloudflare.com/client/v4/organizations/{organization_id}/load_balancers/monitors'  \
+--header 'X-Auth-Email: <EMAIL>' \
+--header 'X-Auth-Key: <API_KEY>' \
+--header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/spectrum/about/load-balancer.md]:
```suggestion
--header "X-Auth-Email: <EMAIL>" \
--header "X-Auth-Key: <API_KEY>" \
--header "Content-Type: application/json" \
```

File: content/ssl/client-certificates/configure-your-mobile-app-or-iot-device.md [hunk 13]
```diff
@@ -195,10 +196,15 @@ $ request_body=$(< <(cat <<EOF
 EOF
 ))
 
-// save the response so we can view it and then extra the certificate
-$ curl -H 'X-Auth-Email: YOUR_EMAIL' -H 'X-Auth-Key: YOUR_API_KEY' -H 'Content-Type: application/json' -d "$request_body" https://api.cloudflare.com/client/v4/zones/YOUR_ZONE_ID/client_certificates > response.json
+# save the response so we can view it and then extra the certificate
+$ curl https://api.cloudflare.com/client/v4/zones/{zone_id/client_certificates \
+--header 'X-Auth-Email: <EMAIL>' \
+--header 'X-Auth-Key: <API_KEY>' \
+--header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/ssl/client-certificates/configure-your-mobile-app-or-iot-device.md]:
```suggestion
--header "X-Auth-Email: <EMAIL>" \
--header "X-Auth-Key: <API_KEY>" \
--header "Content-Type: application/json" \
```

**Review by marciocloudflare — COMMENTED** [review]
just to make sure I don't loose progress so far

File: content/radar/investigate/domain-ranking-datasets.md [hunk 14]
```diff
@@ -89,12 +89,12 @@ If you are interested in a specific top (like the top one million), go through t
...
 ```bash
-curl -X POST "https://api.cloudflare.com/client/v4/radar/datasets/download" \
-     -H "Authorization: Bearer <API_TOKEN>" \
-     -H 'Content-Type: application/json' \
-     --data '{
-      "datasetId": 213
-    }'
+curl "https://api.cloudflare.com/client/v4/radar/datasets/download" \
+--header "Authorization: Bearer <API_TOKEN>" \
+--header 'Content-Type: application/json' \
```
Comment (marciocloudflare) [inline content/radar/investigate/domain-ranking-datasets.md]:
```suggestion
--header "Content-Type: application/json" \
```

**Review by marciocloudflare — APPROVED** [review]

File: content/ssl/client-certificates/configure-your-mobile-app-or-iot-device.md [hunk 15]
```diff
@@ -195,10 +196,15 @@ $ request_body=$(< <(cat <<EOF
 EOF
 ))
 
-// save the response so we can view it and then extra the certificate
-$ curl -H 'X-Auth-Email: YOUR_EMAIL' -H 'X-Auth-Key: YOUR_API_KEY' -H 'Content-Type: application/json' -d "$request_body" https://api.cloudflare.com/client/v4/zones/YOUR_ZONE_ID/client_certificates > response.json
+# save the response so we can view it and then extra the certificate
+$ curl https://api.cloudflare.com/client/v4/zones/{zone_id/client_certificates \
```
Comment (pedrosousa) [inline content/ssl/client-certificates/configure-your-mobile-app-or-iot-device.md]:
```suggestion
$ curl https://api.cloudflare.com/client/v4/zones/{zone_id}/client_certificates \
```

**Pull request merged**